packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -568,3 +569,160 @@ class _TextSelectionHandleOverlayState extends State<_TextSelectionHandleOverlay
     return null;
   }
 }
+
+/// A gesture detector to detect non-exclusive event chains for a text field.
```

Comment:
It seems odd to say a "gesture detector to detect". Maybe: "A gesture detector that matches non-exclusive ...".

Reply:
Changed to respond which is a fairly standard word